php/class-post-type.php

```diff
@@ -316,7 +316,13 @@ public function preserve_post_name_in_insert_data( $post_data, $original_post_da
 	 */
 	public function render_data_metabox( $post ) {
 		$snapshot_content = $this->get_post_content( $post );
-
+		if ( 'publish' !== get_post_status( $post ) ) {
+			$conflicts_settings = $this->get_conflicts_setting( $post );
+			add_thickbox();
+			wp_enqueue_style( 'snapshot-admin' );
```

Comment:
Is this not too late? If it is after `wp_head` does this not mean enqueued styles miss their chance to get printed?